src/main/java/com/breskul/hw/controllers/NasaController.java

```diff
@@ -0,0 +1,78 @@
+package com.breskul.hw.controllers;
+
+import com.breskul.hw.controllers.entity.Photos;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import static org.springframework.http.HttpStatus.MOVED_PERMANENTLY;
+
+@Component
+public class NasaController {
```

Comment:
I think that ``NasaService'' is more suitable for this class in terms of application design. I also would move it to the service folder.